src/scenes/ApplicationRouter/portal/Map/index.tsx

```diff
@@ -120,10 +123,28 @@ const MapView = () => {
   );
 };
 
-const MapViewWrapper = () => (
-  <ApplicationPage>
-    <MapView />
-  </ApplicationPage>
-);
+const MapViewWrapper = () => {
+  const { activeLocale } = useLocalization();
+  const { selectedApplication } = useApplicationData();
+
+  const crumbs: Crumb[] = useMemo(
+    () =>
+      activeLocale && selectedApplication?.id
+        ? [
+            {
+              name: strings.ALL_SECTIONS,
```

Comment:
This is likely a design oversight, but let's make this go back to APPLICATION_PRESCREEN